### ISSUE TITLE
Ethermine is pretending to mine

### ISSUE THREAD

**Issue by pererahiran9** [ethereum-mining/ethminer#651]
So my issue is that my one 1060 6gb rig (my gaming rig that I don't game on anymore) seems to be pretending to mine. What I mean is that I'll leave it on and after some time like 2 hours or something, I'll notice that the GPU temp dropped from its normal 58 C mining temp to 45 C; HOWEVER, ethminer is still running showing a constantly updating hashrate of like 23.9 Mh/s. I also noticed that my graph on ethmine.org for my address shows random drops to 0 Mh/s even at times I know it was running like over night or whatever, and the hashrate goes up at times I know I restarted ethminer. Note though my computer is set to never sleep and this happens while I'm using it too like studying or watching movies on the same rig; I'll just randomly check and see that 45 C and know it's just faking its mining. Then I will have to restart ethminer to get it to actually do something for real because, when I do restart it, it jumps back up to 58 C.

Do you know why it's doing this and how I can stop that? Or maybe someone can teach me how to create a bat file or whatever that will close ethminer after an hour and restart it every hour? (I know almost nothing of computer science.)

**Comment by jean-m-cyr** [comment]
Check your log for errors. 23.9 is high for a 1060, so I'm guessing you're overclocking?

**Comment by pererahiran9** [comment]
Where are these logs? My file for ethminer just has a "bin" folder and inside is the .exe with my start.bat file but nothing else. Also yes I'me overclocking. I've been using stratum and it's not pretending to mine like ethminer is at all so I don't think it has anything to do with the gpu.

**Comment by jean-m-cyr** [comment]
Usually it does have something to do with the GPU. Bet it doesn't happen if you don't overclock.

**Comment by pererahiran9** [comment]
Okay I'll get back to you tomorrow since I don't want to risk this pretending over night. I'm in the process of figuring out how to create those logs. 

Also, don't you think that if Stratum can handle my overclocked GPU running at even higher rates then ethminer should be able to? Is that even heard of for a GPU to crash on ethminer but not on Stratum?

**Comment by Rodriguevb** [comment]
Same error than me. #647 
We need to underclock with the last version :( 
But ethminer never restarts alone like claymore?

**Comment by xstead** [comment]
Just a simple script for keep alive ethminer.
https://github.com/xstead/ethereum-miner-monitor

**Comment by jean-m-cyr** [comment]
@xstead Nice! In your TODO, you have: "monitor ethminer logs"
Why not use the API instead. I believe it would provide you the same info.

**Comment by jean-m-cyr** [comment]
@Rodriguevb 

> We need to underclock with the last version :(

Since when has max overclock become the norm, and we now call less oveclock "underclocking"??? ;-)

**Comment by satori-q3a** [comment]
One of my 1070 is starting to glitch which windows will then do a display reset, which resets the gpu, which resets cuda, which causes ethmine to give an inline error but ignores it and continues 'mining' altho it will never find results after that.

CUDA error in func 'dev::eth::CUDAMiner:: search' at line 489 : unknown error.
 X 12:25:36|cuda-1   Error CUDA mining: unknown error
CUDA error in func 'dev::eth::CUDAMiner:: search' at line 489 : unknown error.
 X 12:25:36|cuda-0   Error CUDA mining: unknown error

This means that the error handler needs work, or at least a running tally of errors in the display would be nice.

**Comment by jean-m-cyr** [comment]
@satori-q3a Sure... I agree. But I don't overclock to the extent I get these errors, and I'm not going to. Waste enough mining time on r&d already, without adding further downtime due to avoidable malfunctions.

**Comment by xstead** [comment]
@jean-m-cyr
Yes, you are right, but I'ld like to monitor directly the log files && analyze ethminer running and working as well or not.

**Comment by RADSOL** [comment]
Having the same issue, only started after going to v13.0. I would prefer if ethminer would just crash instead of continuing to "fake" mine like this. It was mining fine for me for 2 or 3 days before this happened. I'll try to drop the overclock a bit.

**Comment by Xeowolf** [comment]
@satori-q3a 
I was getting the error you reported.
CUDA error in func 'dev::eth::CUDAMiner:: search' at line 489 : unknown error.
X 12:25:36|cuda-1 Error CUDA mining: unknown error

I tried everything I could in software to fix this and nothing worked. (Bios, windows settings, bat file settings, no overclocking etc)
Solution: One of my risers was bad but I had to trouble shoot to figure out which one. Problem solved and its been running over clocked for a bit now.

**Comment by nsmachine** [comment]
I have seen this behavior too. When this happens it just keeps reporting a constant hash rate with no variations. Maybe there are errors reported on the console too but who knows. By the time I realize this has happened it is too late and the only way to know is the graphs on pool stats page fall off.
I thought my memory overclocks were too high or voltage was too low but I ran claymore with the same settings for past 24 hrs and the system was very stable.
I see no hash rate improvement with claymore and would like to use ethminer if it can run with same stability.

I am new to mining. < 2 months so do not have data points to compare the stability against previous versions of ethminer.

**Comment by piotr-dobrogost** [comment]
Pull #688 might help here…

**Comment by clayv** [comment]
This appears to be a duplicate of issue #632 that I opened, but this one getting more comment traction so I am updating comments here.

I was also using GTX 1060 overclocked to +725 and was getting 23.8 MH/sec/card, but after reading @jean-m-cyr comments regarding digtal vs. analog logic (sorry, I can't find that post) I experimented and dropped the OC to +625.  Since making that change, I have not seen the "search" error after running the rig for more than a week and at +725 the rig wouldn't go more than 2 days without experiencing this problem.

No other changes were made to the configuration and the hash rate only dropped to 23.5 MH/sec/card.  Hope this info may help others...

**Comment by piotr-dobrogost** [comment]
Duplicate of issue #632.

**Comment by DeadManWalkingTO** [comment]
After #757 (added --exit parameter to exit whenever an error occurred) you can use a watchdog.

Here is my [ETHminerWatchDogDmW](https://github.com/DeadManWalkingTO/ETHminerWatchDogDmW) Windows7/8/10 [32/64] & Linux (Any Dist/Any Ver/Any Arch) (#735).

Check and feedback please.
Thank you!
